### ISSUE TITLE
doom64.wad md5sum

### ISSUE THREAD

**Issue by MoeFoh** [jnmartin84/doom64-dc#33]
Just bought Doom 64 on Steam ($1.99 special) and I get a md5 of:
0aaba212339c72250f8a53a0a2b6189e doom64.wad

This does not match the md5 stated in the readme.

It does match this v1.1: https://doomwiki.org/wiki/DOOM64.WAD

What's going on?

**Comment by jnmartin84** [comment]
I'm honestly not sure, especially since my file doesn't match either of those. If you can find a way to get me a copy of the file I will see what I can do about supporting it

**Comment by jnmartin84** [comment]
Next time I'm at the computer I'll double-check if I didn't md5sum the EX wad file instead of the remaster WAD

**Comment by jnmartin84** [comment]
Don't attach the file here but if you can send an email/Google drive link or Dropbox link, that would help me help you. Thanks.

**Comment by MoeFoh** [comment]
> Don't attach the file here but if you can send an email/Google drive link or Dropbox link, that would help me help you. Thanks.

What email address should I use?

**Comment by jnmartin84** [comment]
jnmartin84@gmail.com

Thanks, I'll do my best to figure this out

**Comment by jnmartin84** [comment]
After some investigation, I've found the following:
I have never even played the Steam copy I have. I got it as part of a bundle.
If I open it in SLADE, I only see maps 01 through 33.

With the copy that has `0aaba212` md5sum, SLADE will show me maps 34 through 40.

Regardless of those md5sums and WAD contents being different, the maps start at the same locations and convert to the same N64 format MAP WADs, having identical md5sums after the conversion process.

I am going to update README and consider this as resolved/not a bug.
